NitroxServer/GameLogic/GameData.cs

```diff
@@ -0,0 +1,17 @@
+﻿using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using ProtoBufNet;
+using System;
+using System.Collections.Generic;
+
+namespace NitroxServer.GameLogic.Bases
+{
+    [Serializable]
+    [ProtoContract]
+    public class GameData
```

Comment:
I don't think we need this class (we can keep PDASaveData inside World.cs directly for now)

Reply:
i want use this class for all call of general saves

GameData.PDApartial
GameData.pdacomplete

Reply:
I think that makes sense.  However, can we call it either "GameState" or "GameStateData" as all of the other items in PersistedWorldData are game data.

I'll help you find the deconstruction bug.